onnxruntime/core/providers/dml/DmlExecutionProvider/src/Operators/DmlOperatorElementWise.cpp

```diff
@@ -480,7 +480,7 @@ class DmlOperatorElementwisePow : public DmlOperator
         ML_CHECK_VALID_ARGUMENT(kernelInfo.GetOutputCount() == 1);
 
         auto constExpTensor = kernelInfo.TryGetConstantInputTensor(1);
-        if (constExpTensor && constExpTensor->GetTotalElementCount() == 1)
+        if (constExpTensor && constExpTensor->IsCpuData() && constExpTensor->GetTotalElementCount() == 1)
```

Comment:
Since it's an easy mistake to make and we've seen it once before, consider renaming the method to TryGetConstantInputCpuTensor and moving the check for IsCpuData inside it.